docs/architecture/ADR-005-liquidation.md

```diff
@@ -0,0 +1,235 @@
+> 
+> | Borrow Limit (BL) |Borrowed Value (BV) | BV / BL | Liquidation Incentive |
+> | - | - | - | - |
+> | 100 | 100.1 | 1.001 | 0.1% |
+> | 100 | 102 | 1.02 | 2% |
+> | 100 | 105 | 1.05 | 5% |
+> | 100 | 110 | 1.1 | 10% |
+> | 100 | 140 | 1.4 | 10% |
+
+The Dynamic liquidation Incentive option would take advantage of market forces to reduce the liquidation incentive to the lowest sustainable value.
+Liquidators would have a chance to liquidate for a lower incentive (e.g. 2%) if deemed profitable, otherwise the incentive continues to increase as the borrow accrues interest.
+
+The `CloseFactor` parameter can be any value without affecting this method.
+
+In this method, token-specific liquidation incentives (which incentivize liquidation of some asset types before others) would likely modify the parameter `MaxLiquidationIncentive`.
+
+> Option 2: Dynamic close factor
+>
+> Close factor ranges from `0.0 = MinimumCloseFactor` to 1.0 when the borrower is between 0% and `20% = CompleteLiquidationThreshold` over borrow limit, then stays at 1.0
+> 
+> | Borrow Limit (BL) |Borrowed Value (BV) | BV / BL | Close Factor |
+> | - | - | - | - |
+> | 100 | 100.1 | 1.001 | 0.005 |
+> | 100 | 102 | 1.02 | 0.1 |
+> | 100 | 110 | 1.1 | 0.5 |
+> | 100 | 130 | 1.2 | 1.0 |
+> | 100 | 140 | 1.4 | 1.0 |
+
+The Dynamic Close Factor option would again take advantage of market forces to reduce excessive collateral selloffs, by reducing the portion of collateral initially eligible for liquidation.
+Liquidators would have the change to liquidate smaller portions of the borrow if profitable and bring the position back into health.
+Otherwise, close factor would continue to increase as the borrow accrues interest.
+
+This also allows borrows to be liquidated completely in one transaction, once they arr serverely over their borrow limit.
+
+The `LiquidationIncentive` parameter can be any value, including varying from token to token, without affecting this method.
+
+> **++++++++++++++++++**
+> **DECISIONS REQUIRED**
+> **++++++++++++++++++**
+> 
+> Which strategy should we employ for dynamic liquidation parameters
+> 
+> a) No dynamic liquidation. Has parameters:
+>    - Per-token `LiquidationIncentive`
+>    - Global parameter `CloseFactor`
+>
+> b) Dynamic incentive (option 1), with parameters:
+>    - Per-token `MinLiquidationIncentive`
+>    - Per-token `MaxLiquidationIncentive`
+>    - Global parameter `FullLiquidationIncentiveThreshold`
+>    - Global parameter `CloseFactor`
+>
+> c) Dynamic close factor (option 2), with parameters:
+>    - Per-token `LiquidationIncentive`
+>    - Global parameter `MinimumCloseFactor`
+>    - Global parameter `CompleteLiquidationThreshold`
+>
+> Once this has a final decision, I can update the document appropriately.
```

Comment:
Decision required here. It discusses the dynamic liquidation strategies proposed in the alternatives section.

Reply:
Going with option 2 (meeting)